target/linux/realtek/dts-5.15/rtl8393_netgear_gs750e.dts

```diff
@@ -105,9 +105,12 @@
 &ethernet0 {
 	mdio: mdio-bus {
 		compatible = "realtek,rtl838x-mdio";
+		regmap = <&ethernet0>;
 		#address-cells = <1>;
 		#size-cells = <0>;
-		reset-gpios = <&gpio0 23 GPIO_ACTIVE_LOW>;
+		// Switch doesn't come back properly 
+		// after a reset so dont.
```

Comment:
dont -> don't
Also, wouldn't this comment fit on a single (80 char) line?

Reply:
I'm fine with all this, I'll be able to push the changes in a couple days.